File: stud_aid/lib/models/message.dart
import 'package:json_annotation/json_annotation.dart';

part 'message.g.dart';

@JsonSerializable()
class Message {
  int? messageId;
  String? content;
  int? sender;
  int? receiver;

  Message();
  factory Message.fromJson(Map<String, dynamic> json) =>
      _$MessageFromJson(json);

  Map<String, dynamic> toJson() => _$MessageToJson(this);
}

File: StudAidAPI/StudAid.Model/Message.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace StudAid.Model
{
    public partial class Message
    {
        public int MessageId { get; set; }
 
        public string Content { get; set; }
        public int? Sender { get; set; }
        public int? Receiver { get; set; }
    }
}
